lib/PuppeteerSharp/Helpers/AsyncDictionaryHelper.cs

```diff
@@ -58,5 +58,27 @@ internal void AddItem(TKey key, TValue value)
                 tcs.TrySetResult(value);
             }
         }
+
+        internal bool TryRemove(TKey key, out TValue value)
+        {
+            var result = _dictionary.TryRemove(key, out value);
+            _ = _pendingRequests.TryRemove(key, out _);
```

Comment:
Do we need `_ =`?

Reply:
Strictly speaking, no. 
I usually assign to a discard variable to explicitly ignore the return value.